admin/controllers/content/views/order/view.php

```diff
@@ -0,0 +1,151 @@
+<?php
+defined('CMSPATH') or die; // prevent unauthorized access
+
+// TODO: handle custom states - see all view
+// TODO: handle base grid column count - see all view
```

Comment:
its using table, not grid?

Reply:
it's using grid for the trs _inside_ the table